src/rpc/blockchain.cpp

```diff
@@ -2699,6 +2699,91 @@ UniValue CreateUTXOSnapshot(
     return result;
 }
 
+static RPCHelpMan loadtxoutset()
+{
+    return RPCHelpMan{
+        "loadtxoutset",
+        "Load the serialized UTXO set from disk.",
+        {
+            {"path",
+                RPCArg::Type::STR,
+                RPCArg::Optional::NO,
+                "path to the snapshot file. If relative, will be prefixed by datadir."},
+        },
+        RPCResult{
+            RPCResult::Type::ELISION, "", "",
+        },
```

Comment:
In commit "rpc: add loadtxoutset" (82a8744f1384f07dc30aaa6339b0b8fb32dc3af0)

Missing result description